### PR TITLE
CVS-51750 Model zoo custom node use case documentation

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: docs/faces_analysis_dag.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Usage of Object Detection and Recognition models from OpenVINO Model Zoo in DAG
+
+## Analysis of multiple faces in single image frame request
+This document demonstrates how to create complex pipelines using object detection and object recognition models from OpenVINO Model Zoo. As an example, we will use [face-detection-retail-0004](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/face-detection-retail-0004/description/face-detection-retail-0004.md) to detect multiple faces on the image. Then, for each detected face we will crop it using [model_zoo_intel_object_detection](../src/custom_nodes/model_zoo_intel_object_detection) example custom node. Finally, each image face image will be forwarded to [age-gender-recognition-retail-0013](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/age-gender-recognition-retail-0013/description/age-gender-recognition-retail-0013.md) and [emotion-recognition-retail-0003](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/emotions-recognition-retail-0003/description/emotions-recognition-retail-0003.md) models.
+
+![OCR graph](faces_analysis.png)
```
Comment (dtrawins) [inline docs/faces_analysis_dag.md:6]: this is not OCR

File: docs/faces_analysis_dag.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+│   └── 1
+│       ├── face-detection-retail-0004.bin
+│       └── face-detection-retail-0004.xml
+└── lib
+    └── libcustom_node_model_zoo_intel_object_detection.so
+```
+
+## Deploying OVMS
+
+Deploy OVMS with OCR demo pipeline using the following command:
```
Comment (dtrawins) [inline docs/faces_analysis_dag.md:83]: This is not OCR

Comment (dkalinowski) [reply docs/faces_analysis_dag.md]: done

File: docs/faces_analysis_dag.md (hunk 1 above)
Comment (dkalinowski) [reply docs/faces_analysis_dag.md]: done

**Review by dtrawins — APPROVED** [review]

File: docs/faces_analysis_dag.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Using such pipeline, a single request to OVMS can perform a complex set of operations to determine all faces and its properties.
+
+## Pipeline Configuration Graph
+
+Below is depicted graph implementing faces analysis pipeline execution. 
+
+![Faces Analysis Pipeline Graph](faces_analysis_graph.svg)
+
+It includes the following Nodes:
+- Model `face-detection` - inference execution which takes user image as input. It returns outputs including information about face coordinates and confidence levels.
```
Comment (mzegla) [inline docs/faces_analysis_dag.md:17]:
```suggestion
- Model `face-detection` - deep learning model which takes user image as input. Its outputs contain information about face coordinates and confidence levels.
```

File: docs/faces_analysis_dag.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+### Emotions recognition model
+```
+wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/emotions-recognition-retail-0003/FP32/emotions-recognition-retail-0003.xml
+wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/emotions-recognition-retail-0003/FP32/emotions-recognition-retail-0003.bin
+```
+
+## Building the Custom Node "model_zoo_intel_object_detection" Library 
+
+Custom nodes are loaded into OVMS as dynamic library implementing OVMS API from [custom_node_interface.h](../src/custom_node_interface.h).
```
Comment (mzegla) [inline docs/faces_analysis_dag.md:46]:
```suggestion
Custom nodes are loaded into OVMS as dynamic libraries implementing OVMS API from [custom_node_interface.h](../src/custom_node_interface.h).
```

**Review by mzegla — APPROVED** [review]

**Review by atobiszei — APPROVED** [review]

File: docs/faces_analysis_dag.md (hunk 4 above)
Comment (dkalinowski) [reply docs/faces_analysis_dag.md]: Done

File: docs/faces_analysis_dag.md (hunk 3 above)
Comment (dkalinowski) [reply docs/faces_analysis_dag.md]: Done

**Pull request merged**